pkg/runner/conn.go

```diff
@@ -12,22 +11,24 @@ import (
 
 /*
 
-				   Instance model
-				    |          ^
+                   Instance model
+                    |          ^
 functionRequest     |          |   functionResponse
-				    v          |
-			     producer   consumer
-			        |           ^
-	bytes	        |           |     bytes
-					|           ----------------------|
-					|----------------------------> process
+                    v          |
+                 producer   consumer
+                    |           ^
+    bytes	        |           |     bytes
```

Comment:
Not aligned

Reply:
this is a problem, maybe I do find the wrong method to convert tab to space